packages/@aws-cdk/aws-appconfig-alpha/awslint.json

```diff
@@ -26,13 +26,77 @@
         "docs-public-apis:@aws-cdk/aws-appconfig-alpha.IApplication",
 
         "no-unused-type:@aws-cdk/aws-appconfig-alpha.PredefinedDeploymentStrategyId",
-        "ref-via-interface:@aws-cdk/aws-appconfig-alpha.Application.addAgentToEcs.taskDef",
         "props-physical-name:@aws-cdk/aws-appconfig-alpha.ApplicationProps",
         "props-physical-name:@aws-cdk/aws-appconfig-alpha.DeploymentStrategyProps",
         "props-physical-name:@aws-cdk/aws-appconfig-alpha.EnvironmentProps",
         "props-physical-name:@aws-cdk/aws-appconfig-alpha.ExtensionProps",
-        "events-in-interface",
-        "events-method-signature",
-        "events-generic"
+        "events-method-signature:@aws-cdk/aws-appconfig-alpha.Application.on",
+        "events-method-signature:@aws-cdk/aws-appconfig-alpha.Application.onDeploymentBaking",
```

Comment:
@chenjane-dev nit: i know i told you to add each individual linter exemption, because if not, and we stabilized, then we would be exempting `events-method-signature` everywhere. However, I don't think we need the individual `params[0]` and `params[1]` either. Having the `onDeploymentBaking` linter error is enough uniqueness i think

Reply:
I didn't include them at first but then was getting the errors on the `params[0]` and `params[1]`

Reply:
ok i'll check this out for you. seems weird if true but i believe you :)

Reply:
used `*` to get it to work